spec/models/group_invite_status_spec.rb

```diff
@@ -1,59 +1,19 @@
 # frozen_string_literal: true
 
 require "rails_helper"
+require Rails.root.join("spec", "models", "concerns", "setup_status_spec.rb")
 
 RSpec.describe GroupInviteStatus, type: :invite_status do
   subject { GroupInviteStatus }
   let(:organization) { classroom_org }
   let(:grouping)     { create(:grouping, organization: organization) }
-  let(:group)        { Group.create(grouping: grouping, title: "#{Faker::Company.name} Team") }
+  let(:group)        { create(:group, grouping: grouping) }
   let(:invitation)   { create(:group_assignment_invitation) }
 
-  describe "valid", :vcr do
-    let(:invite_status) do
-      subject.create(group: group, group_assignment_invitation: invitation)
-    end
-
-    # TODO: make Group factory so we can make GroupInviteStatus factory to test SetupStatus behavior with:
-    # it_behaves_like 'setup_status'
-
-    it "has a default status of unaccepted" do
-      expect(invite_status.unaccepted?).to be_truthy
-    end
-
-    describe "errored?" do
-      it "is errored? when errored_creating_repo?" do
-        invite_status.errored_creating_repo!
-        expect(invite_status.errored?).to be_truthy
-      end
-
-      it "is errored? when errored_importing_starter_code?" do
-        invite_status.errored_importing_starter_code!
-        expect(invite_status.errored?).to be_truthy
-      end
-    end
+  describe "valid" do
+    let(:invite_status) { create(:group_invite_status, group: group, group_assignment_invitation: invitation) }
 
-    describe "setting_up?" do
-      it "is setting_up? when accepted?" do
-        invite_status.accepted!
-        expect(invite_status.setting_up?).to be_truthy
-      end
-
-      it "is setting_up? when waiting?" do
-        invite_status.waiting!
-        expect(invite_status.setting_up?).to be_truthy
-      end
-
-      it "is setting_up? when creating_repo?" do
-        invite_status.creating_repo!
-        expect(invite_status.setting_up?).to be_truthy
-      end
-
-      it "is setting_up? when errored_importing_starter_code?" do
-        invite_status.importing_starter_code!
-        expect(invite_status.setting_up?).to be_truthy
-      end
-    end
+    it_behaves_like "setup_status"
```

Comment:
All the tests removed in this file are already in `setup_status_spec.rb`. See `it_behaves_like "setup_status"`